src/locale/locales/tr/messages.po

```diff
@@ -167,27 +167,27 @@ msgstr ""
 #. How many days have passed, displayed in a narrow form
 #: src/lib/hooks/useTimeAgo.ts:158
 msgid "{0}d"
-msgstr ""
+msgstr "{0}g"
```

Comment:
For the native turkish speakers:
Bluesky gün kelimesinin tamamı yerine olabildiğince tek harf kullanmamızı istiyor. 

gün yazınca gözüme biraz kötü gözüktü, 1g ile aynı anlam sağlanıyor ve daha güzel gözüküyor diye düşündüğüm için `{0}g` şeklinde commitledim.

![20241117_15h33m16s_grim](https://github.com/user-attachments/assets/d26871c6-fb07-42c2-8729-af0524b4bb1f)
![20241117_15h20m24s_grim](https://github.com/user-attachments/assets/0a266097-819c-4f12-8bde-9f0068e31b17)